pkg/download/automation/automation.go

```diff
@@ -135,11 +134,14 @@ func createTemplateFromRawJSON(obj automationClient.Response, configType string)
 		content = obj.Data
 	}
 	content = jsonutils.MarshalIndent(content)
+	if configType == "workflow" {
```

Comment:
I'm not sure we need to or should limit this to workflows - are we sure that templating is (and will) only be possible for workflows, not other automation resources?

Reply:
I also think that it would be better to apply this to all templates that are produced by the automation interface. Since they have used it already, it can be expected to be expanded to other resources.